docs/router-store/api.md

```diff
@@ -1,5 +1,10 @@
 # API
 
+## Configuration Options
+
+To connect the Angular router module with the NgRx store module, import the store module via
+`StoreRouterConnectingModule.forRoot()`.
+
```

Comment:
What happened to the description for `stateKey`?

Reply:
I'm confused 😅 
Like this you mean?

Reply:
😴 I must have been looking at a partial change. 👍

Reply:
No, I've recommitted. I must have done something wrong...